command/alias/alias.go

```diff
@@ -141,3 +142,19 @@ func NewCommand(home string, docker client.CommonAPIClient, output terminal.Outp
 
 	return cmd
 }
+
+// multipleHostnameValidator validates a comma separated list of hostnames
+type multipleHostnameValidator struct{}
```

Comment:
@marionnewlevant this should move into the `validate` package to keep all of the logic bundled together.

Reply:
@jasonmccallister 
I did think about putting the multipleHostNameValidator in with the other validators. But alias is the only place it is used, and it is important that both the validator and the consumer of the input split on the `,`, and also do the `strings.TrimSpace`. So I would like to either keep the code all in the same file, or else move that parsing into the validator like so:
```
// Parse returns a slice of the hostnames
func (v *MultipleHostNameValidator) Parse(input string) []string, error {...}
```
Options:
- move the validator and duplicate the parsing logic across files
- move the validator and add the complexity of a Parse function
- leave things as they are
- something else I haven't thought of

Your call.

Reply:
Moved the validator and added a Parse function.